src/main/flight/gps_rescue.c

```diff
@@ -768,6 +766,15 @@ void initialiseRescueValues (void)
     rescueState.intent.pitchAngleLimitDeg = 0.0f; // force pitch adjustment to zero - level mode will level out
     rescueState.intent.velocityItermAttenuator = 1.0f; // allow iTerm to accumulate normally unless constrained by IMU error or descent phase
     rescueState.intent.velocityItermRelax = 0.0f; // but don't accumulate any at the start, not until fly home
+    rescueState.intent.targetAltitudeStepCm = 0.0f;
+
+    // get throttle pid coefficients from position_control.c
+    altitudePidCoeffs_t data;
+    getAltitudePidCoeffs(&data);
+    throttlePidCoeffs.kp = data.kp;
+    throttlePidCoeffs.ki = data.ki;
+    throttlePidCoeffs.kd = data.kd;
+    throttlePidCoeffs.kf = data.kf;
```

Comment:
You can just use `getAltitudePidCoeffs()` directly wherever it's needed - I pointed out the necessary change in another comment. Just PM me if you have some issues with handling the pointer that gets returned from the getter.
```suggestion
```

Reply:
implemented this slightly differently, but with similar intent. let me know if it is ok.